test/pjrt/test_profiler.py

```diff
@@ -0,0 +1,59 @@
+import codecs
+import contextlib
+import glob
+import os
+import threading
+
+from absl.testing import absltest
+import torch
+import torch_xla.core.xla_model as xm
+import torch_xla.experimental.pjrt as pjrt
+import torch_xla.debug.profiler as xp
+
+
+@contextlib.contextmanager
+def _profile(logdir: str, port: int = 9012):
+  server = xp.start_server(port)
+
+  tracer = threading.Thread(target=xp.trace, args=(f'localhost:{port}', logdir))
+  tracer.setDaemon(True)
+  tracer.start()
+  yield
+
+
+class TestPjRtProfiler(absltest.TestCase):
+
+  def setUp(self):
+    assert pjrt.using_pjrt()
+
+    # HACK: ensure libtpu is loaded if using TPU
+    xm.xla_device()
```

Comment:
Is there a more explicit way to initialize the ComputationClient?

Reply:
I think this is fine, this is how I usually trigger computationClient creation too..

Reply:
would this test work on non-tpu platform?

Reply:
Yes. I tested it on CPU and it works, although it's too flaky to add to the CI (see my comments below)